Garllon/graphql_connector
# Support mutations

Adds `mutation` in order to allow such graphql queries

### Where to start review
[`query_builder.rb`](https://github.com/Garllon/graphql_connector/pull/21/files#diff-cbf47143b4cc8f82c47c334917a945df)
### Manual testing
* Start a graphql service that has a mutation type
* Configure `gem` as described in README to point to that server
* Use newly introduced `mutation` feature and expect to succeed
* Test also with `service class inclusion` as described in README
